full-node/db/sov-db/src/schema/tables.rs

```diff
@@ -168,12 +176,14 @@ macro_rules! define_table_with_default_codec {
 /// little-endian, but RocksDB uses lexicographic ordering which is only
 /// compatible with big-endian, so we use [`bincode`] with the big-endian option
 /// here.
+// TODO `bincode` is expected to have `no-std` soon; should be usable under `no-std`
+#[cfg(feature = "std")]
```

Comment:
From what I see all the `cfg(feature = "std")]` pragmas in this file are on functions signatures. Would it be possible to move all the `std` code to another file and reimport it as:
```
 `cfg(feature = "std")]`
 mod table_moule;
  `cfg(feature = "std")]`
  pub use table_moule::*;
```
And inside `tables_std` we won't need   `cfg(feature = "std")]`